odoo/tools/cache.py

```diff
@@ -162,26 +164,60 @@ def determine_key(self) -> None:
 def log_ormcache_stats(sig=None, frame=None):   # noqa: ARG001 (arguments are there for signals)
     """ Log statistics of ormcache usage by database, model, and method. """
     from odoo.modules.registry import Registry
-    cache_entries = {}
-    current_db = None
-    cache_stats = ['Caches stats:']
-    for (dbname, model, method), stat in sorted(STAT.items(), key=lambda k: (k[0][0] or '~', k[0][1], k[0][2].__name__)):
-        dbname_display = dbname or "<no_db>"
-        if current_db != dbname_display:
-            current_db = dbname_display
-            cache_stats.append(f"Database {dbname_display}")
-        if dbname:   # mainly for MockPool
-            if (dbname, stat.cache_name) not in cache_entries:
-                cache = Registry.registries.d[dbname]._Registry__caches[stat.cache_name]
-                cache_entries[dbname, stat.cache_name] = Counter(k[:2] for k in cache.d)
-            nb_entries = cache_entries[dbname, stat.cache_name][model, method]
-        else:
-            nb_entries = 0
-        cache_name = stat.cache_name.rjust(25)
-        cache_stats.append(
-            f"{cache_name}, {nb_entries:6d} entries, {stat.hit:6d} hit, {stat.miss:6d} miss, {stat.err:6d} err, {stat.gen_time:10.3f}s time, {stat.ratio:6.1f}% ratio for {model}.{method.__name__}"
-        )
-    _logger.info('\n'.join(cache_stats))
+    log_msgs = ['Caches stats:']
+    # {dbname: [fp_entries, nb_entries]}
+    db_stats = defaultdict(lambda: [0, 0])
+    # {dbname: {(model, method): [fp_entries, nb_entries, stat]}}
+    cache_stats = defaultdict(lambda: defaultdict(lambda: [0, 0, None]))
+    
+    column_info = (
+        f"{'Cache Name':>25},"
+        f"{'Entries':>15},"
+        f"{'Memory [Bytes]':>19},"
```

Comment:
Could you add maximum memory used by an entry?
```suggestion
        f"{'Memory [sum]':>19},"
        f"{'Memory [max]':>19},"
```